### ISSUE TITLE
OSX Scan Wrong Mac Address

### ISSUE THREAD

**Issue by fabiovaz** [geolado/Default-Password-Wifi-Owner#21]
While trying get the macaddress only returns: "-84", "-78", "-48", etc.. not the real macaddress

**Comment by caioluders** [comment]
Yeah, I think something changed on osx command output, will investigate.

**Comment by fabiovaz** [comment]
/System/Library/PrivateFrameworks/Apple80211.framework/Versions/A/Resources/airport no more  send MAC Address, maybe need some external library tike spoof-mac.
